orchagent/dash/dashorch.h

```diff
@@ -17,12 +17,16 @@
 #include "timer.h"
 #include "zmqorch.h"
 #include "zmqserver.h"
+#include "flex_counter_manager.h"
 
 #include "dash_api/appliance.pb.h"
 #include "dash_api/route_type.pb.h"
 #include "dash_api/eni.pb.h"
 #include "dash_api/qos.pb.h"
 
+#define ENI_STAT_COUNTER_FLEX_COUNTER_GROUP "ENI_STAT_COUNTER"
+#define ENI_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS 10000
```

Comment:
How did we decide the default polling interval as 10 sec. Is it based on some spec?

Reply:
Every other object other than PORT has a default of 10 sec. you think i should update this?